NugetTemplates/SharpArch.NHibernate/SharpArch.NHibernate.nuspec

```diff
@@ -13,9 +13,11 @@
     <summary>Architectural foundation for building maintainable applications with Domain Driven Design patterns and practices</summary>
     <language>en-US</language>
     <dependencies>
-	  <dependency id="SharpArch.Domain" version="(1.9.6,3.0.0)" />
-      <dependency id="FluentNhibernate" version="[1.4,2.0)" />
-      <dependency id="Microsoft.AspNet.Mvc" version="[4.0.20710,4.0.30506.0]" />
+	  <dependency id="SharpArch.Domain" version="[2.4.0,3.0.0)" />
+      <dependency id="FluentNhibernate" version="[2.0.1.0,3.0.0.0)" />
+      <dependency id="Microsoft.AspNet.Mvc" version="[5.2.2,6.0.0)" />
+      <dependency id="NHibernate.Validator" version="[1.3.2.0,2.0.0.0)" />
```

Comment:
How come we are introducing NHibernate.Validator again? If I remember correctly, we don't use it any more.